### DISCUSSION TITLE
播放界面无法隐藏

### DISCUSSION THREAD

**Discussion by Prnceton** [hooke007/mpv_PlayKit#478] [mpv 通用]
应该跟下载的片源有关系，播放的时候左上角一直显示标题和章节，右边音量调节滑块，下面所有按钮全都一直显示，找不到隐藏的办法
<img width="1920" alt="Snipaste" src="https://github.com/hooke007/MPV_lazy/assets/40190995/dd9eb863-547d-44c9-b068-a21d3dc2b058">

**Comment by hooke007** [comment]
请发送片段或出处以供我验证

**Reply by Prnceton** [reply]
> 请发送片段或出处以供我验证

上传了一个体积小一点的，不过也有32G

**Reply by hooke007** [reply]
![image](https://github.com/hooke007/MPV_lazy/assets/41094733/99522a6a-119d-47a6-be19-38a08d3c2998)
你这个样品没发现问题

**Reply by Prnceton** [reply]
你全屏能把所有按钮都隐藏吗？

我把鼠标放到最下面，会变成这样
![Aquaman](https://github.com/hooke007/MPV_lazy/assets/40190995/b940f178-5e50-49e9-84ba-699bcc515df4)

如果把鼠标移上来，会变成这样
![Aquaman2](https://github.com/hooke007/MPV_lazy/assets/40190995/16309b1e-aa44-491a-93b3-88a5fdddd7f5)

**Reply by hooke007** [reply]
我这里没有任何问题

**Reply by hooke007** [reply]
你是在自己配置还是在使用懒人包

**Reply by Prnceton** [reply]
有没有隐藏界面所有按钮的操作方法，其他片源打开就自动隐藏了

**Reply by Prnceton** [reply]
> 你是在自己配置还是在使用懒人包

懒人包，带vsMega版本

**Reply by hooke007** [reply]
那你是改动过什么吗，为什么连缩略图都没显示

本身就是鼠标不靠近就不显示，你这个bug我只觉得莫名其妙

**Reply by Prnceton** [reply]
没改动，我又重新安装了下，还是有些片源打开自动隐藏界面，有些片源打开不隐藏，靠近章节也不显示缩略图

**Reply by hooke007** [reply]
请上传日志供我进一步分析

**Reply by Prnceton** [reply]
怎么上传日志

**Reply by hooke007** [reply]
开启这个选项就能生成日志
https://github.com/hooke007/MPV_lazy/blob/9713164e5bd2342c55c15902c690fd8095393aa0/portable_config/mpv.conf#L9
![image](https://github.com/hooke007/MPV_lazy/assets/41094733/9df45d31-30f8-4a73-9556-481790dada62)

**Reply by hooke007** [reply]
另，如果这个选项也无法强制跟随鼠标一起隐藏，那暂时无解
https://github.com/hooke007/MPV_lazy/blob/d06c7b8b1716cf6ad754fda14cf10ceba5faea1c/portable_config/script-opts.conf#L506

**Reply by Prnceton** [reply]
看了两个片源，Aquaman不能隐藏，Dune能自动隐藏
[mpv-lazy.log](https://github.com/user-attachments/files/16099053/mpv-lazy.log)

script-opts-append = uosc-autohide=no这个yes和no都试了没用

**Reply by hooke007** [reply]
更新一下mpv本体可以解决，问题具体在哪我也不知道

**Comment by clspeter** [comment]
遇到相同問題，就是某些片源會無法隱藏。更新mpv本體解決了，謝謝！
